Add support for VTOL multivehicle simulation in SITL gazebo

Jaeyoung-Lim opened the pull request:
**Describe problem solved by this pull request**
This adds support to multivehicle simulation with `standard_vtol` by adding a xacro file which generates the sdf models

**Describe your solution**
Generate the sdf file of `standard_vtol` using xacro so that mavlink port / sysid can be configured

**Test data / coverage**
![ezgif com-video-to-gif(2)](https://user-images.githubusercontent.com/5248102/72689212-57189b00-3b0f-11ea-82aa-bdbd13ce5353.gif)

Full video: [video](https://youtu.be/lAjjTFFZebI)

To run this, you can run with the following command
```
Tools/gazebo_sitl_multiple_run.sh -m standard_vtol
```

**Additional Context**
The motor / motor columns are not rendered since xacro inside px4 sitl doesn't seem to let you have multiple visual elements within the same link

TSC21 commented on models/rotors_description/urdf/vtol_base.xacro:
```diff
@@ -0,0 +1,114 @@
+            <mesh scale="0.001 0.001 0.001" filename="model://standard_vtol/meshes/x8_wing.dae" />
+        </geometry>
+      </visual>
+      <!-- <visual name='left_motor_column'>
```
Do we really need all this commented code here?

Jaeyoung-Lim replied:
No, it was commented out from the original sdf file since xacro doesn't seem to be able to handle multiple visual elements within a link. Removed now!

Jaeyoung-Lim committed:
Generate vtol models from xacro

Added fixedwing_base.xacro


wip


wip fixedwing xacro


Fix


Getting close(er)


imported iris


Finally something starts to work


Added control surfaces


Cleanup fixedwing_base_macro


More cleanups


Clean clean clean


Final cleanup


Control surfaces moving


Add standard vtol model


Fix


Fix


Fix


Add elevator


wip vtol


Fix


Some more fixes


Align control surfaces


Add fixed wing flight


Support multiple vtol in gazebo


Remove comments
